cl/lib/solr_core_admin.py

```diff
@@ -10,9 +10,10 @@
 from cl.lib.sunburnt import SolrError
 
 
-def create_temp_solr_core(core_name, schema_path, delete_if_present=True,
+def create_temp_solr_core(core_name, core_source_path, schema_path,
```

Comment:
Do you want to fix this up in a separate PR before or revert your changes to this file? It feels weird to have changes here *except* to adapt to the new work you did in freelawproject/solr?

Reply:
Yes. This was not meant to be in this PR.